web/src/utils/money.test.ts

```diff
@@ -0,0 +1,60 @@
+/**
```

Comment:
nice, the first test!

Reply:
Good job! Consider putting the test in a test-specific folder (web/test/utils/money.test.ts) to avoid cluttering with the source code; but no hurry doing that right now; can do it later when you have more tests coming along.

Reply:
@xlai20  Right now I decided to move them into the same folder to co-locate them, in this manner: https://medium.com/trabe/colocating-react-component-files-the-tools-you-need-c377a61382d3
I think colocating is pretty nice and we do not have to re-order the test folder structure whenever we make changes to the source code folder structure. And as long as tests are named `*.test.{ts | tsx}` they will run when we do `yarn test`, so we do not have to choose the specific `test/` folder at root to run tests.
But if you think that a root `test/*` folder is better we can always refactor the tests out in future PRs.

Reply:
Sure, currently the test sitting here is fine. Let's make decision after the entire folder structure is more stabilized then.